vpn_slice/linux.py

```diff
@@ -93,6 +93,10 @@ def deconfigure_firewall(self, device):
 
 
 class CheckTunDevProvider(TunnelPrepProvider):
+    def create_tunnel(self):
+        if not os.path.exists('/dev/net/tun'):
+            subprocess.check_call(['mkdir', '-p', '/dev/net'])
```

Comment:
Let's please use the built-in `os` functions here, since external tools aren't necessary:

```python
# need import stat
os.makedirs('/dev/net', exist_ok=True)
os.mknod(path, mode=0o640 | os.stat.S_IFCHR, device = os.makedev(10, 200))
```